test/replication-status.test.js

```diff
@@ -89,8 +89,9 @@ Object.keys(testAPIs).forEach(API => {
     })
 
     it('has correct replication info after loading from snapshot', async () => {
+      await db._cache._store.open();
       await db.saveSnapshot()
-      await db.close()
+      // await db.close()
```

Comment:
I'm unsure if we can remove this. The test here is supposed to "load the db form a snapshot" (as in "fresh"), so having everything closed as a starting point is probably better (in order to make sure it all works). We could possible move the `saveSnapshot` call to a `before(() => ...)` to make that clear?